fargate/log-puller.yml

```diff
@@ -205,6 +217,64 @@ Resources:
       ListenerArn:
         Fn::ImportValue:
           !Join [':', [!Ref 'StackName', 'PublicListener443']]
-      Priority: !Ref 'Priority'
+      Priority: !Ref 'PriorityForPath'
+
+  LoadBalancerRule80Host:
```

Comment:
I might be missing something, but it might be better to put in place an automatic redirect listener. Like this: https://github.com/concord-consortium/cloud-formation/blob/master/lara-ecs.yml#L263-L278

Then I don't think you'll need these extra `80` rules.  I guess that listener would be added to the stack from which the PublicListener80 is being imported.

Reply:
Won't that cause a web redirect to app.concord[qa].org losing the hostname (which I thought Cynthia cared about)?

Reply:
The redirect config linked above is supposed to include the host name in the redirect. We'd have to check to make sure but I think it will work.